### DISCUSSION TITLE
Welcome to hoverzoom discussions!

### DISCUSSION THREAD

**Discussion by extesy** [extesy/hoverzoom#637] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GrosPoulet** [comment]
Hi Hoverzoom fans!
I started contributing to this awesome project back in 2016.
I would like this extension to be adopted by more and more users because it really makes web surfing a much deeper travel.
I'll release some major improvements 🎁 in coming weeks, so stay tuned 😉
